pa11y_scripts/.without-report

```diff
@@ -0,0 +1,24 @@
+{
+    "urls": [{
+            "url": "http://127.0.0.1:8000/",
```

Comment:
Do we know if the url is fully loaded after this point? Should we wait for an element on this page to be visible?

Reply:
pa11y waits for headless chrome to open and load the provided URL before running. We only need a wait action if we're interacting with the page, navigating to a new page, or there's some other kind of page modifications _after_ the page has been loaded.

Reply:
Thanks, works for me.